loris/transforms.py

```diff
@@ -268,6 +263,12 @@ def _scales_to_reduce_arg(self, image_request, image_info):
             arg = str(reduce_arg)
         return arg
 
+    def _create_fifo(self, tmp):
+        fifo_fp = path.join(tmp, 'fifo.bmp')
+        mkfifo_call = '%s %s' % (self.mkfifo, fifo_fp)
+        subprocess.check_call(mkfifo_call.split())
```

Comment:
Rather than joining these into a string and then splitting them, could we pass them as the list that `check_call` expects? i.e.

```python
mkfifo_call = [self.mkfifo, fifo_fp]
subprocess.check_call(mkfifo_call)
```